### PR TITLE
add wifi setup, fix logos with transparent spaces

### LINKED ISSUE #7: When not using 'empty_as_transparent' = True, logos are cut off
When not using 'empty_as_transparent' = True, logos are cut off in the middle of the screen. @the-papi

### PR DESCRIPTION
**Pull request by gorrdy**
demo https://www.youtube.com/watch?v=m6Be1W4tS_0, now it is a bit different but the idea remains.

### CODE REVIEW

**Comment by gorrdy** [comment]
also fixes #7

File: MOONCLOCK/font.py [hunk 1]
```diff
@@ -15,6 +15,9 @@
 CHAR_WIDESATOSHI = '\ue011'
 CHAR_MONEY_BAG = '\ue012'
 CHAR_HALVING = '\ue013'
+CHAR_WIFI = '\ue014'
+CHAR_CHECK = '\ue015'
+CHAR_CROSS = '\ue016'
```
Comment (the-papi) [inline MOONCLOCK/font.py]: That doesn't seem to be the correct hexadecimal counting :smile:

7, 8, 9, a, b, c, d, e, f, 11, 12, ....

File: MOONCLOCK/code.py [hunk 2]
```diff
@@ -49,33 +49,45 @@
...
     fail_count = 0
     for wifi_conf in secrets:
         try:
             print('Connecting to {}'.format(wifi_conf['ssid']))
-            display_group.render_string('connecting', center=True)
-            display_group.show()
-            time.sleep(1)
-            display_group.render_string('to ' + wifi_conf['ssid'], center=True)
+            display_group.clear()
+            display_group.render_string('{0}{1}{2}'.format(font.CHAR_WIFI, ' ', wifi_conf['ssid'][:8]), center=False)
```
Comment (the-papi) [inline MOONCLOCK/code.py]: `'{} {}'.format(font.CHAR_WIFI, wifi_conf['ssid'][:8])` is better

File: MOONCLOCK/code.py [hunk 3]
```diff
@@ -49,33 +49,45 @@
...
             display_group.show()
             time.sleep(1)
             wifi.radio.connect(wifi_conf['ssid'], wifi_conf['password'])
             print('Connected to {}!'.format(wifi_conf['ssid']))
             print('My IP address is', wifi.radio.ipv4_address)
-            display_group.render_string('Connected! ', center=True)
+            display_group.clear()
+            display_group.render_string('{0}{1}'.format(font.CHAR_CHECK, ' '), center=True)
```
Comment (the-papi) [inline MOONCLOCK/code.py]: `'{} '.format(font.CHAR_CHECK)`

**Review by the-papi — CHANGES_REQUESTED** [review]

**Comment by the-papi** [comment]
Maybe we should also keep backwards compatibility with the old `secrets.py` file?

Something like...
```python
if isinstance(secrets, dict):
    secrets = [secrets]
```

**Review by the-papi — APPROVED** [review]

**Pull request merged**